sharktank/sharktank/models/flux/flux.py

```diff
@@ -155,19 +175,19 @@ def __init__(self, theta: Theta, params: FluxParams):
         self.pe_embedder = EmbedND(
             dim=pe_dim, theta=params.theta, axes_dim=params.axes_dim
         )
-        self.add_module("img_in", LinearLayer(theta("img_in")))
-        self.add_module("time_in", MLPEmbedder(theta("time_in")))
-        self.add_module("vector_in", MLPEmbedder(theta("vector_in")))
+        self.add_module("img_in", LinearLayer(self.theta("img_in")))
+        self.add_module("time_in", MLPEmbedder(self.theta("time_in")))
```

Comment:
No need to change this, but for my edification: Does adding `self` ever matter? We should probably avoid modifying the theta once its being handed to constructors.

Reply:
Now it is allowed to init without a theta as the theta could be retrieved from the config in a standard way in the `ThetaLayer` base. This allows us to construct with only a config.